src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Components/TimeDateSettings.cs

```diff
@@ -95,6 +100,12 @@ internal static List<PluginAdditionalOption> GetAdditionalOptions()
                     Value = true,
                 },
                 new PluginAdditionalOption()
+                {
+                    Key = nameof(GlobalQueryIgnoreNumberInput),
+                    DisplayLabel = Resources.Microsoft_plugin_timedate_SettingGlobalQueryIgnoreNumberInput,
+                    Value = false,
```

Comment:
Setting is default off as I think this makes sense and to not make unit tests to complicated